runelite-client/src/main/java/net/runelite/client/plugins/runenergy/RunEnergyOverlay.java

```diff
@@ -0,0 +1,225 @@
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.tooltip.Tooltip;
+import net.runelite.client.ui.overlay.tooltip.TooltipManager;
+import org.apache.commons.lang3.StringUtils;
+
+public class RunEnergyOverlay extends Overlay
+{
+	private static final ImmutableSet ALL_GRACEFUL_HOODS = ImmutableSet.of(
+		GRACEFUL_HOOD, GRACEFUL_HOOD_11851, GRACEFUL_HOOD_13579, GRACEFUL_HOOD_13580,
+		GRACEFUL_HOOD_13591, GRACEFUL_HOOD_13592, GRACEFUL_HOOD_13603, GRACEFUL_HOOD_13604,
+		GRACEFUL_HOOD_13615, GRACEFUL_HOOD_13616, GRACEFUL_HOOD_13627, GRACEFUL_HOOD_13628,
+		GRACEFUL_HOOD_13667, GRACEFUL_HOOD_13668, GRACEFUL_HOOD_21061, GRACEFUL_HOOD_21063
+	);
+
+	private static final ImmutableSet ALL_GRACEFUL_TOPS = ImmutableSet.of(
+		GRACEFUL_TOP, GRACEFUL_TOP_11855, GRACEFUL_TOP_13583, GRACEFUL_TOP_13584,
+		GRACEFUL_TOP_13595, GRACEFUL_TOP_13596, GRACEFUL_TOP_13607, GRACEFUL_TOP_13608,
+		GRACEFUL_TOP_13619, GRACEFUL_TOP_13620, GRACEFUL_TOP_13631, GRACEFUL_TOP_13632,
+		GRACEFUL_TOP_13671, GRACEFUL_TOP_13672, GRACEFUL_TOP_21067, GRACEFUL_TOP_21069
+	);
+
+	private static final ImmutableSet ALL_GRACEFUL_LEGS = ImmutableSet.of(
+		GRACEFUL_LEGS, GRACEFUL_LEGS_11857, GRACEFUL_LEGS_13585, GRACEFUL_LEGS_13586,
+		GRACEFUL_LEGS_13597, GRACEFUL_LEGS_13598, GRACEFUL_LEGS_13609, GRACEFUL_LEGS_13610,
+		GRACEFUL_LEGS_13621, GRACEFUL_LEGS_13622, GRACEFUL_LEGS_13633, GRACEFUL_LEGS_13634,
+		GRACEFUL_LEGS_13673, GRACEFUL_LEGS_13674, GRACEFUL_LEGS_21070, GRACEFUL_LEGS_21072
+	);
+
+	private static final ImmutableSet ALL_GRACEFUL_GLOVES = ImmutableSet.of(
+		GRACEFUL_GLOVES, GRACEFUL_GLOVES_11859, GRACEFUL_GLOVES_13587, GRACEFUL_GLOVES_13588,
+		GRACEFUL_GLOVES_13599, GRACEFUL_GLOVES_13600, GRACEFUL_GLOVES_13611, GRACEFUL_GLOVES_13612,
+		GRACEFUL_GLOVES_13623, GRACEFUL_GLOVES_13624, GRACEFUL_GLOVES_13635, GRACEFUL_GLOVES_13636,
+		GRACEFUL_GLOVES_13675, GRACEFUL_GLOVES_13676, GRACEFUL_GLOVES_21073, GRACEFUL_GLOVES_21075
+	);
+
+	private static final ImmutableSet ALL_GRACEFUL_BOOTS = ImmutableSet.of(
+		GRACEFUL_BOOTS, GRACEFUL_BOOTS_11861, GRACEFUL_BOOTS_13589, GRACEFUL_BOOTS_13590,
+		GRACEFUL_BOOTS_13601, GRACEFUL_BOOTS_13602, GRACEFUL_BOOTS_13613, GRACEFUL_BOOTS_13614,
+		GRACEFUL_BOOTS_13625, GRACEFUL_BOOTS_13626, GRACEFUL_BOOTS_13637, GRACEFUL_BOOTS_13638,
+		GRACEFUL_BOOTS_13677, GRACEFUL_BOOTS_13678, GRACEFUL_BOOTS_21076, GRACEFUL_BOOTS_21078
+	);
+
+	// Agility skill capes also count for the Graceful set effect
+	private static final ImmutableSet ALL_GRACEFUL_CAPES = ImmutableSet.of(
+		GRACEFUL_CAPE, GRACEFUL_CAPE_11853, GRACEFUL_CAPE_13581, GRACEFUL_CAPE_13582,
+		GRACEFUL_CAPE_13593, GRACEFUL_CAPE_13594, GRACEFUL_CAPE_13605, GRACEFUL_CAPE_13606,
+		GRACEFUL_CAPE_13617, GRACEFUL_CAPE_13618, GRACEFUL_CAPE_13629, GRACEFUL_CAPE_13630,
+		GRACEFUL_CAPE_13669, GRACEFUL_CAPE_13670, GRACEFUL_CAPE_21064, GRACEFUL_CAPE_21066,
+		AGILITY_CAPE, AGILITY_CAPET, AGILITY_CAPE_10648, AGILITY_CAPE_13340, AGILITY_CAPET_13341
```

Comment:
Also max cape

Reply:
Good catch! This only includes the non-cosmetic versions, right? (e.g. doesn't include the max fire cape, for example)

Reply:
Ye. Also, why are there 42 different agility capes? Are you sure some of them aren't mounted versions or something?

Reply:
Is there any way to check?

Reply:
Don't think so no, but it looks like `AGILITY_CAPE` and `AGILITY_CAPET` are the correct items

Reply:
Okay, cheers! I'm also not sure about the huge amount of Graceful item variations but I'm unsure of how to check those. :(

Reply:
Ignore that comment lol